Libraries/Modal/NativeModalManager.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ * @format
+ */
+
+'use strict';
+
+import type {TurboModule} from 'RCTExport';
+import * as TurboModuleRegistry from 'TurboModuleRegistry';
+
+export interface Spec extends TurboModule {
+  // RCTEventEmitter
+  +addListener: (eventName: string, handler: Function) => Object;
```

Comment:
remove these listeners, since it is not an event emitter

Reply:
RCTModalManager actually extends RCTEventEmitter

Reply:
What @fkgozali said I couldn't get flow to run otherwise.